envknob/features.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package envknob
+
+import (
+	"errors"
+	"runtime"
+
+	"tailscale.com/version"
+	"tailscale.com/version/distro"
+)
+
+// CanRunTailscaleSSH reports whether serving a Tailscale SSH server is
+// supported for the current os/distro.
+func CanRunTailscaleSSH() error {
```

Comment:
envknob seems like an unusual place for this, since it's not actually a knob that you can configure.  Would putting it in `ssh/tailssh` make sense?  That way, I would expect it to stay more up to date with any changes in our SSH support.  Most of the files in that package are using build tags, so probably a new file that applies to all platforms?

Reply:
oh, got it.  that makes sense then.